jetbrains-core/src/software/aws/toolkits/jetbrains/services/sqs/actions/DeleteQueueAction.kt

```diff
@@ -23,4 +23,7 @@ class DeleteQueueAction : DeleteResourceAction<SqsQueueNode>(message("sqs.delete
         client.deleteQueue { it.queueUrl(selected.queueUrl) }
         project.refreshAwsTree(SqsResources.LIST_QUEUE_URLS)
     }
+
+    override val displayComment: Boolean = true
+    override val comment: String = message("resource.delete.warning_text", "queue")
```

Comment:
we don't need the boolean since the presence of a comment implies that it should be displayed